.eslintrc.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
     ],
     "@typescript-eslint/no-floating-promises": "error",
   },
-  ignorePatterns: ["packages/cli/", "packages/generator/templates"],
+  ignorePatterns: ["packages/cli/", "packages/generator/templates", ".eslintrc.js"],
```

Comment:
I'm just curious, what was the problem that we need to add this?

Reply:
Eslint would fail to start linting the codebase unless `.eslintrc.js` was marked as ignored.
I think the error has something to do with this breaking change: https://eslint.org/docs/user-guide/migrating-to-7.0.0#runtime-deprecation-warnings-for-eslintrc-config-files